test/helper.lua

```diff
@@ -33,19 +62,75 @@ function helpers.create_space_with_tree_index(engine)
     end
     local space = create_space(space_name, engine)
 
-    space:create_index("primary", {type = "TREE", parts={ 1 }})
-    space:create_index("index_for_first_name", {type = "TREE", parts={ 2 }})
-    space:create_index("multipart_index", {type = "TREE", parts={ {3, is_nullable = true}, {4, is_nullable = true} }})
-    space:create_index("non_unique_index", {type = "TREE", parts={ {5, is_nullable = true} }, unique = false})
+    space:create_index("primary", {
+        type = "TREE",
+        parts = {
+            1
+        }
+    })
+    space:create_index("index_for_first_name", {
+        type = "TREE",
+        parts = {
+            2
+        }
+    })
+    space:create_index("multipart_index", {
+        type = "TREE",
+        parts = {
+            {
+                3,
+                is_nullable = true
+            },
+            {
+                4,
+                is_nullable = true
```

Comment:
nit: The mix of array and map looks ugly. Could you please change it to {field = 4, is_nullable = true}.

Reply:
Agree, fixed it.